tip-wlan/resources/environments/dev-amazon-tip.yaml

```diff
@@ -19,22 +19,17 @@ global:
   nsPrefix: tip
```

Comment:
IS this `dev-amazon-tip.yaml` file is used by someone or just a template? I'm asking, because this PR adds some specific configuration values like ACM cert ids, dns names, etc... If this file is a template, we might want to create a new one that is more specific. WDYT?

Reply:
for now it's used by other aws deployments (cicd and qa clusters)

Reply:
cleaned up from deployment specific details